added an entry for getUTCSeconds term (#2494)
Codecademy/docs: content/javascript/concepts/dates/terms/getUTCSeconds/getUTCSeconds.md

---
Title: '.getUTCSeconds()'
Description: 'Returns the seconds (0 to 59) of a date according to universal time.'
Subjects:
  - 'Web Development'
  - 'Computer Science'
Tags:
  - 'Date'
  - 'Methods'
CatalogContent:
  - 'introduction-to-javascript'
  - 'paths/front-end-developer-career-path'
---

> The getUTCSeconds() method returns the seconds (ranging from 0 to 59) of a specific date according to UTC (Coordinated Universal Time).

Reviewer: ```suggestion
The **`getUTCSeconds()`** method returns the seconds (ranging from 0 to 59) of a specific date according to UTC (Coordinated Universal Time).
```

## Syntax

> 

Reviewer: Only need one line break here.
```suggestion
```

```pseudo
Date.getUTCSeconds()
```

A `Date` object is used for working with dates and times. The object provides a way to create, manipulate, and format dates in various formats.

A `Date` object can be created by declaring a new variable with the `new Date()` constructor.

## Example

> Consider the following example that uses the `.getUTCSeconds()` method to get the seconds of `2023-06-06T10:30:04` and logs the result to the console:

Reviewer: ```suggestion
The following example uses the `.getUTCSeconds()` method to get the seconds of `2023-06-06T10:30:04` and logs the result to the console:
```

```js
const date = new Date('2023-06-06T10:30:04');
const seconds = date.getUTCSeconds();

console.log(seconds);
```

This example results in the following output:

```shell
4
```

> The example below demonstrates the use of `.getUTCSeconds()` to print the current time in seconds to the console:

Reviewer: Let's add a line break and make this a codebyte example.
```suggestion

## Codebyte Example

The codebyte example below uses the `.getUTCSeconds()` method to get the current time in seconds and logs it to the console:
```

> ```js

Reviewer: ```suggestion
```codebyte/javascript
```

const today = new Date();
const today_seconds = today.getUTCSeconds();

> console.log(today_seconds) //ouput: current second of todays UTC time

Reviewer: ```suggestion
console.log(today_seconds); 
```

---
Title: '.getUTCSeconds()'
Description: 'Returns the seconds (0 to 59) of a date according to universal time.'
Subjects:
  - 'Web Development'
  - 'Computer Science'
Tags:
  - 'Date'
  - 'Methods'
CatalogContent:
  - 'introduction-to-javascript'
  - 'paths/front-end-developer-career-path'
---

The **`getUTCSeconds()`** method returns the seconds (ranging from 0 to 59) of a specific date according to UTC (Coordinated Universal Time).

> 

Reviewer: ```suggestion
```
